### ISSUE TITLE
Bad order in Print one at time mode

### ISSUE THREAD

**Issue by Zogar89** [Ultimaker/Cura#9662]
[Orden incorrecto.zip](https://github.com/Ultimaker/Cura/files/6361653/Orden.incorrecto.zip)
### Application Version

4.9

### Platform

Windows 10

### Printer

Artillery X1

### Reproduction steps

1-Put print One at Time Mode
2-Order objects over print bed.
3-Layer Start X/Y = 0

### Actual results

Print order for 0X 0Y coordenates.
![imagen](https://user-images.githubusercontent.com/11767960/115794178-edd3a780-a3a3-11eb-9a91-5875891f9ba3.png)
![imagen](https://user-images.githubusercontent.com/11767960/115794224-03e16800-a3a4-11eb-86f9-d4c35c4b3245.png)
![imagen](https://user-images.githubusercontent.com/11767960/115794270-165ba180-a3a4-11eb-8fb1-4e8e4865f0d7.png)
![imagen](https://user-images.githubusercontent.com/11767960/115794290-22476380-a3a4-11eb-9d0b-846b75763f11.png)
![imagen](https://user-images.githubusercontent.com/11767960/115794317-312e1600-a3a4-11eb-9232-9e0049587111.png)

### Expected results

Good print order. From 0X 0Y

**Comment by Zogar89** [comment]
Update!

I have reviewed the Layer Start Coordinates option and understand that CURA when positioning objects on different layers, this option is invalid.
The solution I have found is to reposition the objects manually.

As I understand that this is not a bug but that the program changes its way of working, I propose that in the mode one at a time, perhaps in the list of objects, the objects could be ordered or the order of printing be represented with a number or some similar solution.
